colossalai/fx/_compatibility.py

```diff
@@ -0,0 +1,42 @@
+from typing import Callable
+
+import torch
+
+from . import META_COMPATIBILITY
```

Comment:
should we keep this `META_COMPATIBILITY` constant within the `_compatiblity.py` instead of `fx.__init__.py`?

Reply:
I see